template/package.json

```diff
@@ -3,31 +3,6 @@
     "version": "0.1.0",
```

Comment:
We should rename this package as "babylonjs-editor-templates" (with the s) because it contains multiple templates

Reply:
for it i need to change folder name from template to templates as i understand, but we have templates folder for our zip templates, which name should be for this folder?